bin/dev-tools/create_shipments.py

```diff
@@ -0,0 +1,332 @@
+        shipper_response = self._parse_request(
+            f"{self.profiles_url}/api/v1/wallet/generate",
+            headers={'Authorization': 'JWT {}'.format(self.shipper['token'])})
+        carrier_response = self._parse_request(
+            f"{self.profiles_url}/api/v1/wallet/generate",
+            headers={'Authorization': 'JWT {}'.format(self.carrier['token'])})
+        if not shipper_response or not carrier_response:
+            if self.verbose:
+                print(f'Error generating {"shipper" if not shipper_response else "carrier"} wallet.')
+
+            self._process_failure(f'Error generating wallets')
+
+        if self.moderator:
+            moderator_response = self._parse_request(
+                f"{self.profiles_url}/api/v1/wallet/generate/",
+                headers={'Authorization': 'JWT {}'.format(self.moderator['token'])})
+            if not moderator_response:
+                if self.verbose:
+                    print(f'Error generating moderator wallet.')
+                self._process_failure(f'Error generating moderator wallets')
+
+            self.attributes['moderator_wallet_id'] = moderator_response['data']['id']
+        self.attributes['shipper_wallet_id'] = shipper_response['data']['id']
+        self.attributes['carrier_wallet_id'] = carrier_response['data']['id']
+
+    def _set_storage_credentials(self):
+        if self.verbose:
+            print(f'Creating storage credentials.')
+
+        response = self._parse_request(
+            f"{self.profiles_url}/api/v1/storage_credentials", data={
+                'driver_type': 'local',
+                'base_path': '/shipments',
+                'options': "{}",
+                'title': f'Shipment creator SC: {str(uuid4())}'
+            }, headers={'Authorization': 'JWT {}'.format(self.shipper['token'])})
+
+        if not response:
+            self._process_failure(f'Error generating storage credentials')
+
+        if self.verbose:
+            print(f'Created storage credentials: {response["data"]["id"]}')
+        self.attributes['storage_credentials_id'] = response['data']['id']
+
+    # pylint:disable=unused-variable
+    def _add_telemetry(self, device_id):
+        if self.verbose:
+            print(f'Creating sensors for device: {device_id}')
+        telemetry_data = []
+        self.telemetry_data['device_id'] = device_id
+        for i in range(3):
+            attributes = {
+                'name': f'Sensor: {str(uuid4())}',
+                'hardware_id': f'Hardware_id: {str(uuid4())}',
+                'units': 'c',
+            }
+            response = self._parse_request(
+                f'{self.profiles_url}/api/v1/device/{device_id}/sensor/',
+                data=attributes,
+                headers={'Authorization': 'JWT {}'.format(self.shipper['token'])}
+            )
+            if not response:
+                print(f'{"Failed to create" if not response.ok else "created"} sensor for device: {device_id}')
```

Comment:
If you hit this line your program will likely throw a `NameError`.  The only way in here is `if not response` and then you check for a property on `response`.